src/main/java/com/github/onsdigital/thetrain/configuration/AppConfiguration.java

```diff
@@ -147,18 +174,63 @@ private static Path loadWebsitePathConfig() throws ConfigurationException {
     }
 
     private static int loadPublishPoolSizeConfig() throws ConfigurationException {
+        String value = System.getenv(MAX_REQUEST_SIZE_MB);
         try {
             return Integer.parseInt(System.getenv(THREAD_POOL_SIZE_ENV_KEY));
-        } catch (NumberFormatException e) {
-            throw new ConfigurationException(THREAD_POOL_SIZE_PARSE_ERR, e);
+        } catch (NumberFormatException ex) {
+            throw new ConfigurationException(formatParsingError(THREAD_POOL_SIZE_ENV_KEY, value, Integer.class), ex);
         }
     }
 
     private static int loadPortConfig() throws ConfigurationException {
+        String value = System.getenv(MAX_REQUEST_SIZE_MB);
         try {
             return Integer.parseInt(System.getenv(PORT_ENV_KEY));
         } catch (NumberFormatException e) {
-            throw new ConfigurationException(PORT_PARSE_ERR, e);
+            throw new ConfigurationException(formatParsingError(PORT_ENV_KEY, value, Integer.class), e);
+        }
+    }
+
+    private static Path createTmpFileUploadsDir() throws ConfigurationException {
+        try {
+            Path p = Files.createTempDirectory("tmp");
+            p.toFile().deleteOnExit();
+            return p;
+        } catch (Exception ex) {
+            throw new ConfigurationException("error creating tmp file uploads dir", ex);
         }
     }
-}
+
+    private static long loadMaxFileSizeConfig() throws ConfigurationException {
+        String value = System.getenv(MAX_REQUEST_SIZE_MB);
```

Comment:
```suggestion
        String value = loadMaxRequestSizeConfig();
```
Consider using the existing function where you are getting the default `MAX_REQUEST_SIZE_MB `, so that you get some extra validation / safety checking